apps/docs/pages/guides/integrations/prisma.mdx

```diff
@@ -153,18 +143,17 @@ Update your `.env` file with the following changes:
 1. Rename the `DATABASE_URL` environment variable to `DIRECT_URL`
 1. Create a `DATABASE_URL` environment variable and paste in the new connection string from the dashboard as its value
 
-It is recommended to minimize the number of concurrent connections by setting the `connection_limit` to `1`. You can set connection limit by appending `?connection_limit=1` to your connection string
+Append the `?pgbouncer=true` flag to the `DATABASE_URL` variable.  You can set connection limit by appending `connection_limit=1` to your connection string.
 
 Your `.env` file should resemble the following:
 
 ```env
 # .env
 # PostgreSQL connection string used for migrations
-DIRECT_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres"
+DIRECT_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:5432/postgres"
 # PostgreSQL connection string with pgBouncer config — used by Prisma Client
-DATABASE_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres?connection_limit=1"
+DATABASE_URL="postgres://postgres:[YOUR-PASSWORD]@db.[YOUR-PROJECT-REF].supabase.co:6543/postgres?pgbouncer=true&connection_limit=1"
```

Comment:
The `connection_limit=1` here is an anti pattern that will make the app performance a lot worse. It should only be recommended in specific cases where it is required.